src/coins.h

```diff
@@ -97,6 +97,11 @@ class CCoins
     //! as new tx version will probably only be introduced at certain heights
     int nVersion;
 
+    //| If coins comes from a certificate, nFirstBwtPos represents the position of the first backward transfer;
+    //! All outputs after nFirstBwtPos, including nFirstBwtPos, are backward transfers
+    //! If coins comes from a tx, it is currently set to zero and not used
```

Comment:
Fix the comment; moreover a static const in NOT_VALID_POS = -1 could be used

Reply:
Done